packages/actions/src/helpers/security.ts

```diff
@@ -0,0 +1,38 @@
+import fetch from "node-fetch"
```

Comment:
I think the whole thing should be moved to the `phase2cli` package as the package `actions` should have nothing to do with Github directly. In fact, if you notice in the `actions` package even the functions for handshaking with Firebase use generic OAuth credentials that are a superset of Github's.

Reply:
What about keeping it in the actions, and also provide some basic checks like this ones for other OAuth2 providers that one might use with the mpc-tool (for instance Twitter -> e.g. check it has at least 2 real followers and 1 post). This could be utilities that coordinators decide to use and if so they can easily import in the same way. What do you think?

Reply:
If you want to keep them in the actions, you should provide agnosticism (e.g., the method should not be based on "2" real followers and "1" post, but on "x" real followers and "y" post, tl;dr everything should be parameterized); otherwise, more refactoring will be necessary